CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ pull request if there was one.
 
 ### Changed:
 
+- [Making Custom Sketch operations possible in PostAggreations](https://github.com/yahoo/fili/issues/740)
+    * Removed `FuzzySetPostAggregation asSketchEstimate(SketchAggregation candidate)` method from `FieldConverters` interface.
+    * Added `ThetaSketchEstimatePostAggregation asSketchEstimate(Aggregation candidate)` and `ThetaSketchEstimatePostAggregation asSketchEstimate(PostAggregation postAggregation)` to the class `ThetaSketchFieldConverter`.  
```

Comment:
I assume each line in changelog file should also not be longer than 120 chars. @michael-mclawhorn What do you think?